### DISCUSSION TITLE
Complete options to control Pagination style in all carousel type widgets.

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21235] [Feature Request]
### Prerequisites

- [ ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team @avivu and @nicholaszein👋

As you know, Elementor now has limited options for customizing pagination in carousels.

![pagination](https://user-images.githubusercontent.com/88034902/216765398-f5e4ff4c-a789-429d-aab4-439b1de9326b.png)

I wanted to request that you please provide full pagination customization options for future widgets.

### Describe the solution you'd like

Possibility to customize pagination in **normal, hover and active modes**. this is fabulous. Just like jet woobuilder, which has this capability :

![jet woo](https://user-images.githubusercontent.com/88034902/216765939-3bb0de3b-ee99-4769-b1be-5a5bc9bf6442.png)

This feature allows us to experience unlimited customization and even implement [such a thing](https://prnt.sc/gFqMcu2YnQfT).

In general, paging requires the following options (in three modes: normal, hover and active) :

- color
- width
- height
- border
- Border Radius
- box shadow

Also, the spacing option will be great for setting the distance between [pagination Dots](https://prnt.sc/dsYhZlqgeObk).

**Comment by nicholaszein** [comment]
This discussion was mentioned in:

- #21290
